sotodlib/preprocess/processes.py

```diff
@@ -244,6 +244,35 @@ class Demodulate(_Preprocess):
     def process(self, aman, proc_aman):
         hwp.demod_tod(aman, **self.process_cfgs)
 
+
+class Glitchfill(_Preprocess):
```

Comment:
Sorry, nitpicking now -- looks like the naming convention for preprocess modules is "ThingAction", so I would think this would be GlitchFill.